### PR TITLE
Add Netdata tutorial

### PR DESCRIPTION
**Pull request by satrobit**
Hi!
As the title suggests, This is a tutorial on how to install and secure Netdata.

### CODE REVIEW

File: tutorials/install-secure-netdata/01.en.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+In this tutorial, we're going to install the Netdata and learn how to prevent public access to its web interface since Netdata doesn't provide authentication by itself.
+
+**Prerequisites**
+
+* A fresh CentOS or Ubuntu installation (Almost all mainstream distributions are supported though)
+* Make sure you have root access to the server
+
+## Step 1 - Install and configure Netdata
+### Step 1.1 - Installation
+In order to install the latest version of Netdata, you can use this bash script provided by Netdata itself. Simply run the following in your terminal:
```
Comment (n-se) [inline tutorials/install-secure-netdata/01.en.md:31]: can you link https://learn.netdata.cloud/docs/agent/packaging/installer/#alternative-methods here?

File: tutorials/install-secure-netdata/01.en.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+Redhat/CentOS/Fedora:
+```bash
+sudo yum install nginx
+```
+Your NGINX setup should be good to go.
+
+### Step 2.2 - Setup Authentication
+Run this command to create a username-password pair:
+```bash
+sudo htpasswd -c /etc/nginx/.htpasswd user1
```
Comment (n-se) [inline tutorials/install-secure-netdata/01.en.md:85]: I dont think htpasswd is installed by default, you probably need to install `apache2-utils` before

File: tutorials/install-secure-netdata/01.en.md [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+Add the following lines into your `http` block:
+
+```
+upstream backend {
+   server 127.0.0.1:19999;
+   keepalive 64;
+}
+
+server {
+   listen <10.0.0.1>:19999;
```
Comment (n-se) [inline tutorials/install-secure-netdata/01.en.md:106]: Maybe you can do with a domain and port 80?

**Comment by satrobit** [comment]
Hi @n-se .
Thank you for your suggestions. I've made some changes according to your notes.

**Pull request merged**